fendermint/vm/topdown/src/finality/fetch.rs

```diff
@@ -34,6 +34,12 @@ macro_rules! retry {
                 tracing::warn!(
                     "cannot query ipc parent_client due to: {e}, retires: {retries}, wait: {wait:?}"
                 );
+
+                // there is no point in retrying if the current block is null round
+                if is_null_round_error(&e) {
```

Comment:
Should this be moved above the warn, so it doesn't suggest there will be retries?

Reply:
I shifted the logging